CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # `linya` Changelog
 
+## 0.3.0 (2022-03-08)
```

Comment:
Let's label this `Unreleased` for now. This isn't a breaking change, right? So I might end up just bumping the `PATCH` number.